backend/curriculum_tracking/tests/test_BurndownSnapshot.py

```diff
@@ -64,3 +64,71 @@ def test_calculations_correct(self):
         self.assertEqual(snapshot.project_cards_total_count, 2)
         self.assertEqual(snapshot.cards_in_complete_column_total_count, 2)
         self.assertEqual(snapshot.project_cards_in_complete_column_total_count, 1)
+
+    def test_snapshot_saves_latest_data(self):
+
+        factories.AgileCardFactory(
+            content_item=factories.ContentItemFactory(
+                content_type=ContentItem.PROJECT,
+                project_submission_type=ContentItem.LINK,
+            ),
+            assignees=[self.user],
+        )
+        factories.AgileCardFactory(
+            content_item=factories.ContentItemFactory(
+                content_type=ContentItem.PROJECT,
+                project_submission_type=ContentItem.LINK,
+            ),
+            assignees=[self.user],
+            status=AgileCard.COMPLETE,
+        )
+
+        BurndownSnapshot.create_snapshot(user=self.user)
+        snapshot = BurndownSnapshot.objects.first()
+        snapshot.save()
+
+
+        self.assertEqual(snapshot.user, self.user)
+        self.assertEqual(snapshot.cards_total_count, 2)
+        self.assertEqual(snapshot.project_cards_total_count, 2)
+        self.assertEqual(snapshot.cards_in_complete_column_total_count, 1)
+        self.assertEqual(snapshot.project_cards_in_complete_column_total_count, 1)
+
+        factories.AgileCardFactory(
+            content_item=factories.ContentItemFactory(content_type=ContentItem.TOPIC),
+            assignees=[self.user],
+        )
+
+        BurndownSnapshot.create_snapshot(user=self.user)
+        BurndownSnapshot.create_snapshot(user=self.user)
+
+        factories.AgileCardFactory(
+            content_item=factories.ContentItemFactory(content_type=ContentItem.TOPIC),
+            assignees=[self.user],
+            status=AgileCard.COMPLETE,
+        )
+
+        BurndownSnapshot.create_snapshot(user=self.user)
+        snapshot = BurndownSnapshot.objects.first()
+        snapshot.save()
```

Comment:
this version of the snapshot variable doesn't get used before you create the next one so you can probably delete